gandalf.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.